med-inventory/src/components/purchase-medicine/purchase-layout/numericEditor.jsx

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from "react";
+
+const KEY_BACKSPACE = 8;
+const KEY_DELETE = 46;
+const KEY_F2 = 113;
+
+export default class NumericEditor extends Component {
```

Comment:
Can we move this component to "Shared", considering reusability and functionally meaningless positioning it in the purchase-medicine

Reply:
Will move it